master.js

```diff
@@ -1007,15 +962,74 @@ async function getPlugins(){
 		let plugin = plugins[i];
 		if(plugin.main.onLoadFinish && typeof plugin.main.onLoadFinish == "function") await plugin.main.onLoadFinish({plugins});
 	}
-	console.log("All plugins loaded in "+(Date.now() - startPluginLoad)+"ms");
 	return plugins;
 }
 
+async function startServer() {
+	// Set the process title, shows up as the title of the CMD window on windows
+	// and as the process name in ps/top on linux.
+	process.title = "clusterioMaster";
+
+	// add better stack traces on promise rejection
+	process.on('unhandledRejection', r => console.log(r));
 
-masterPlugins = await getPlugins();
+	/** Sync */
+	function randomStringAsBase64Url(size) {
+	  return base64url(crypto.randomBytes(size));
+	}
+
+	if (!fs.existsSync("secret-api-token.txt")) {
+		config.masterAuthSecret = randomStringAsBase64Url(256);
+		fs.writeFileSync("config.json",JSON.stringify(config, null, 4));
```

Comment:
Had a thought while reviewing - we have all the file names hardcoded in the code right now with no ability to pass config file name when starting the process and all other file paths can actually be defined in the config file - that way you can use the same install with multiple files. But more importantly - no hard coded file names in the code all over the place.
Thoughts?

Reply:
I think secret-api-token.txt is kind of a weird way to do it.  Would much rather have a command line interface to generate a token of desired time length.  But also this is kind of out of scope for this PR.